test/abilities/unburden.test.ts

```diff
@@ -365,7 +365,7 @@ describe("Abilities - Unburden", () => {
       .battleType("double")
       .enemyMoveset([MoveId.SPLASH, MoveId.THIEF])
       .moveset([MoveId.SPLASH, MoveId.REVIVAL_BLESSING])
-      .startingHeldItems([{ name: "WIDE_LENS" }]);
+      .startingHeldItems([{ name: "GRIP_CLAW" }]);
```

Comment:
```suggestion
      .startingHeldItems([{ name: "LUCKY_EGG" }]);
```
(Just in case Grip Claw's item-related interactions interfere with the test)